src/containers/MapContainer/RightCards/Actions.styles.ts

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+import { theme, devices } from 'src/styles'
+
+export const ActionsContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+  @media ${devices.desktop} {
+    margin: ${theme.spacing(2, 4)};
+  }
+`
+
+export const ReopenContainer = styled.div`
+  @media ${devices.desktop} {
+    margin: ${theme.spacing(2, 4)};
+  }
+`
+
+export const NoActionsSpan = styled.span`
```

Comment:
Pas d'indication de balisage dans les noms des composants

```suggestion
export const NoActions = styled.span`
```